cpu/atmega_common/cpu.c

```diff
@@ -66,9 +73,11 @@ void _reset_cause(void)
             DEBUG("Watchdog reset!\n");
         }
     }
+#if !defined (CPU_ATMEGA328P)
     if (mcusr_mirror & (1 << JTRF)) {
         DEBUG("JTAG reset!\n");
     }
+#endif
 }
 
 void cpu_init(void)
```

Comment:
mega-xplained is still reboot-looping even with ```wdt_reset();``` and ```wdt_disable();```. Not sure why. Anyone else tried reboot using the shell?

Reply:
I'll try on my side, at least with uno and mega2560 I don't experience reboot loops.

Reply:
Looks like it's working now on mega-xplained, at last!